packages/ui/src/components/Modal/style.less

```diff
@@ -0,0 +1,97 @@
+@import "antd/lib/modal/style/index.less";
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+
+@condo-modal-margin-vertical: @condo-global-spacing-40;
+@condo-modal-margin-horizontal: @condo-global-spacing-40;
+@condo-modal-border-width: @condo-global-border-width-default;
+@condo-modal-padding: @condo-global-spacing-40;
+@condo-modal-half-padding: (@condo-modal-padding / 2);
+@condo-modal-half-padding-without-border: @condo-modal-half-padding - @condo-modal-border-width;
+@condo-modal-close-icon-width: 24px;
+@condo-modal-close-icon-height: 24px;
+@condo-modal-close-icon-top: @condo-modal-padding +
+  ((@condo-global-line-height-h3 - @condo-modal-close-icon-height) / 2);
+@condo-modal-header-padding-right: (@condo-modal-padding * 2) + @condo-modal-close-icon-width;
+@condo-modal-header-scrolling-border-bottom-color: @condo-global-color-gray-3;
+@condo-modal-footer-buttons-gap: @condo-global-spacing-20;
+
+.condo-modal-root {
+  .condo-modal-mask {
+    background-color: @condo-global-color-gray-7;
+    opacity: 0.5;
+  }
+
+  .condo-modal {
+    max-width: calc(100vw - @condo-modal-margin-horizontal * 2);
```

Comment:
why we need to use too many new CSS features?

Reply:
Probably you need to use ANT styles and then wverride some of them

Reply:
This is the style override.
Only we replace the prefix "ant" with the prefix "condo"

Reply:
`calc` is old AF and `@var * x` is default less feature, so nothing criminal here 👍